custom/list.libsonnet

```
@@ -0,0 +1,14 @@
+{
+  core+: {
+    v1+: {
+      list: {
+        new(items):: {
+          apiVersion: 'v1',
+        } + {
+          kind: 'List',
```

I guess this is taken from elsewhere, but it is a bit pointless to split this object here :)

```suggestion
        new(items):: {
          apiVersion: 'v1',
          kind: 'List',
```